app.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "MusicStore",
+    "description": "ASP.NET 5 sample app",
+    "logo": "https://raw.githubusercontent.com/aspnet/MusicStore/master/src/MusicStore/wwwroot/Images/home-showcase.png",
```

Comment:
@friism wondering if it is possible to give a relative url here? Since this contains `master` everytime when code moves from one branch to another this will have to be fixed.